src/main/java/net/dv8tion/jda/api/events/channel/update/ChannelUpdateTypeEvent.java

```diff
@@ -23,10 +23,16 @@
 
 import javax.annotation.Nonnull;
 
-//TODO-v5: Docs
+/**
+ * Indicates that a {@link Channel Channels} {@link ChannelType} was updated.
```

Comment:
This should mention that this is for seeing when a Text -> News and News -> Text